tensorboard/uploader/uploader.py

```diff
@@ -594,6 +622,7 @@ def __init__(
         rpc_rate_limiter,
         max_request_size,
         max_tensor_point_size,
+        tracker=None,
```

Comment:
Don't default to None, now that this is required

Reply:
Done. Same for another line like this.